tester/i2c.go

```diff
@@ -0,0 +1,52 @@
+package tester
+
+import (
+	qt "github.com/frankban/quicktest"
+)
+
+// I2CBus implements the I2C interface in memory for testing.
+type I2CBus struct {
+	C       *qt.C
```

Comment:
Again, I'd use an interface type and unexport this.

Reply:
Done.